features/external_tables.md

```diff
@@ -56,3 +57,90 @@ On many Postgres providers, you can also set up this connection in the other dir
 ## Other External Table Engines
 
 Hydra is working on providing connections to other data sources in the near future. If you have a request, please reach out to us via our support channel.
+
+## MySQL External Tables
+
+MySQL External Tables are implemented using [`mysql_fdw`](https://github.com/EnterpriseDB/mysql_fdw).
+To set up a MySQL External Table from `psql`, run the following SQL, replacing `...` with your server's information.
+
+```sql
+CREATE EXTENSION mysql_fdw;
+
+CREATE SERVER remote_mysql_server FOREIGN DATA WRAPPER mysql_fdw OPTIONS (host '...', port '...');
+
+CREATE USER MAPPING FOR CURRENT_USER SERVER remote_mysql_server OPTIONS (username '...', password '...');
+```
+
+Same as `postgres_fdw`, you can now create foreign tables from your remote MySQL database.
+Assuming your remote MySQL database's name is `mysql`, you can [import all of your remote tables](https://www.postgresql.org/docs/current/sql-importforeignschema.html) at once as follows:
+
+```sql
+CREATE SCHEMA remote_mysql;
+
+IMPORT FOREIGN SCHEMA mysql FROM SERVER remote_mysql_server INTO remote_mysql;
+```
+
+## HTTP External Sources
```

Comment:
While HTTP is different than other ETs, do we want to introduce a new term? You can create a very similar effect to External Tables using a view, but it is read-only... so, I might suggest an example where you create a view.

Reply:
Added an example that use a view with the `http` ext: https://github.com/HydrasDB/docs/pull/7/commits/ec51b3e24385083162dcfd0f0025cc8435866c8f